### ISSUE TITLE
Invalid offence with 'Unnecessary spacing detected' on assert_difference

### ISSUE THREAD

**Issue by anderkonzen** [bbatsov/rubocop#2320]
After migrating from 0.31.0 to 0.34.2, I'm getting the following error:

````
test/controllers/v1/promos_controller_test.rb:28:39: C: Unnecessary spacing detected.
      assert_difference 'Promo.count', +1 do
                                      ^
````

I don't see why a space there would hurt. Is this correct?

Rubocop version:
0.34.2 (using Parser 2.2.3.0, running on ruby 2.2.2 x86_64-linux)

**Comment by jonas054** [comment]
That's a bug in `ExtraSpacing`. It's fixed in #2298 but not released yet.

**Comment by anderkonzen** [comment]
Nice, thanks!
